src/coreclr/src/jit/codegenarmarch.cpp

```diff
@@ -189,6 +189,13 @@ void CodeGen::genCodeForTreeNode(GenTree* treeNode)
             genCodeForNegNot(treeNode);
             break;
 
+#if defined(TARGET_64BIT)
```

Comment:
Seems wrong to use `TARGET_64BIT` here instead of `TARGET_ARM64`

Reply:
@BruceForstall well, this file has 6 `TARGET_64BIT` usages so I didn't know which one to use. `TARGET_64BIT` is defined for ARM64.
So should I replace it with `TARGET_ARM64` ?

Reply:
The current cases are related to 64-bitness, such as dealing with `long` type on 32-bit machines.

In your case, you create the nodes under `TARGET_ARM64` so they should be consumed under the same `#ifdef`.

This is all minor, since the effect is the same, here.